cli/commands/db/diff.ts

```diff
@@ -0,0 +1,37 @@
+import { Change, diffLines } from 'diff'
+import { DB_MANIFEST } from '../../manifests'
+import { CONNECTION_NAME } from '../../lib/db-utils'
+import {
+  generateScript,
+  getAllConnections,
+  printScriptDiff,
+} from '../../lib/utils'
+
+export default async function run() {
+  // NB: Connection is a term-of-art in Auth0 for an identity provider,
+  // including the username/password database
+  const connections = await getAllConnections()
+  const dbConnection = connections.find((conn) => conn.name === CONNECTION_NAME)
+  if (!dbConnection) {
+    throw new Error('Could not find requested Auth0 connection')
+  }
+  const diffs: [DBActionScriptDefinition, Change[]][] = []
+  for (const scriptDef of DB_MANIFEST) {
+    const our_script = await generateScript(scriptDef, 'db')
+    const existing_script = dbConnection.options.customScripts?.[scriptDef.name]
+    if (!existing_script) {
```

Comment:
Maybe this is the correct behaviour, although it seems like maybe we instead just want to add the script to a `missingScripts` var, and note that the script is missing. E.g. if we're deploying this to a new tenant, the scripts might not be there initially, or maybe we've manually added one of them etc...

Reply:
Auth0 prefills them when you turn on the correct settings.